pkg/server/registry/service.go

```diff
@@ -53,6 +56,14 @@ func (s *LocalContainerRegistry) Start() error {
 		return err
 	}
 
+	if _, err := exec.LookPath("docker"); err != nil {
+		log.Fatal("Cannot find Docker. Please install and try again.")
+	}
+
+	if err := exec.Command("docker", "info").Run(); err != nil {
```

Comment:
You don't need to use `exec`, you can use the `dockerClient` from below and do `dockerClient.Info()`

Reply:
done